### DISCUSSION TITLE
ComboBox (Autocomplete) enable <Home> and <End> keys with <Shift> to select the text

### DISCUSSION THREAD

**Discussion by garnerp** [tailwindlabs/headlessui#1133] [Ideas]
Would it be possible to add <HOME> and <END> keys in the ComboBox (Autocomplete), and with the <SHIFT> to select.  Its a keyboard shortcut that is used regularly to select text in an entry field.

**Comment by chris-b1** [comment]
Alternatively / additionally, I think it would be nice to have an option to opt-out of the `<Home>`/`<End>` behavior all together, as it least to me it feels non-standard, compared to other combo box / command palette UIs (e.g. VS Code)

**Comment by simeydk** [comment]
I expected the input component to behave like a normal `input` component, where <kbd>HOME</kbd>, <kbd>END</kbd>, <kbd>SHIFT</kbd>+<kbd>HOME</kbd> and <kbd>SHIFT</kbd>+<kbd>END</kbd> work as in all text editing tools. 

However, I see for `Combobox`, this is configured to jump to the start or end of the list of available options. This may make sense if the user expects to be in "toggling between options" mode, rather than "typing mode". Does the component have a concept of this difference? Or else is there a way to opt out of this behavious as @chris-b1 requested above?
